### DISCUSSION TITLE
Multi Route – Want to show multiple tooltips with a single route.

### DISCUSSION THREAD

**Discussion by Rajatkumarbehera** [gilbarbara/react-joyride#987] [Q&A]
Hello, @gilbarbara !!!

Can we show **multiple tooltips with a single route**?
If yes, could you please guide me how can we do this?

Thank you.

**Comment by gilbarbara** [comment]
Hey @Rajatkumarbehera 

You can't do that with react-joyride. It's based on multiple steps with a single target per step.
However, it uses the [react-floater](https://github.com/gilbarbara/react-floater) library to render the steps, and you could use it to render multiple tooltips simultaneously.

**Reply by yadavalok1** [reply]
Hey @gilbarbara , can you provide a demo code for this , that will be really helpful for beginners. Thanks :)

**Reply by gilbarbara** [reply]
Hey @yadavalok1 
There's a [demo](https://codesandbox.io/s/github/gilbarbara/react-floater/tree/main/demo) link in the react-floter README

**Reply by yadavalok1** [reply]
Thanks for the quick response @gilbarbara. I am using react joyride and trying to do something like this. 

`<JoyRide
        {...tourState}
        callback={callback}
        showSkipButton={true}
        scrollToFirstStep={true}  
        styles={{
          tooltipContainer: {
            textAlign: "left",
          },
          buttonBack: {
            marginRight: 10,
          },
          options: {
            zIndex: 2000,
          },
          floaterProps: {
            styles: {
              arrow: {
                color: "red",
              }
            }
          },
        }}
        locale={{
          last: "End tour",
        }}
      >
      </JoyRide>`

But it seems the floaterProps is not working and after some debugging I got [this line of code](https://github.com/gilbarbara/react-joyride/blob/6c84d3e8ff10d5f4acc21bee6ee1f544f9b94c0a/src/modules/step.ts#L71) that may be responsible. I am not sure , so can you please guide me on this?

**Reply by gilbarbara** [reply]
Is this related to multi-route? 
Not sure I understand your problem...
Is the arrow color? If it is, check the styling [docs](https://docs.react-joyride.com/styling).

**Reply by yadavalok1** [reply]
Yeah I want to customize the arrow style, like the length, width but I just found arrowColor to change color and also I got the floaterProps but still the customization is limited , so is there any way of customizing the arrow as per our need or only that much customization we can do?

**Reply by gilbarbara** [reply]
So, it's not multi-route related...
Yeah, you can change the length, spread, and some other CSS properties, like color or border, but there's no way to change the SVG data.
